packages/wonder-blocks-color/src/__docs__/color.stories.mdx

```diff
@@ -0,0 +1,234 @@
+import {Meta, Subtitle, ColorPalette, ColorItem} from "@storybook/addon-docs";
+import {StyleSheet} from "aphrodite";
+
+import Color from "@khanacademy/wonder-blocks-color";
+
+import ComponentInfo from "../../../../.storybook/components/component-info.js";
+import {name, version} from "../../package.json";
+
+import {Swatch} from "../docutils/swatch.js";
+
+<Meta
+    title="Color / Color"
+    parameters={{
+        previewTabs: {
+            canvas: {hidden: true},
+        },
+        viewMode: "docs",
+    }}
+/>
+
+export const ProductColors = {
+    blue: Color.blue,
+    purple: Color.purple,
+    gold: Color.gold,
+    green: Color.green,
+    red: Color.red,
+};
+
+export const NeutralColors = {
+    offBlack: Color.offBlack,
+    offBlack64: Color.offBlack64,
+    offBlack50: Color.offBlack50,
+    offBlack32: Color.offBlack32,
+    offBlack16: Color.offBlack16,
+};
+
+export const WhiteShades = {
+    offBlack8: Color.offBlack8,
+    offWhite: Color.offWhite,
+    offWhite64: Color.offWhite64,
+    offWhite50: Color.offWhite50,
+    white: Color.white,
+};
+
+export const BrandColors = {
+    darkBlue: Color.darkBlue,
+    lightBlue: Color.lightBlue,
+    teal: Color.teal,
+    pink: Color.pink,
+};
+
+# Color
+
+<Subtitle>
+    <ComponentInfo name={name} version={version} />
+</Subtitle>
+
+## Summary
+
+<ColorPalette>
+    <ColorItem
+        title="Product colors"
+        subtitle="Use only accessible colors against appropriate backgrounds for links, buttons, and other navigational elements."
+        colors={ProductColors}
+    />
+    <ColorItem
+        title="Neutral colors"
+        subtitle="Used for text and chrome components."
+        colors={NeutralColors}
+    />
+    <ColorItem colors={WhiteShades} />
+    <ColorItem
+        title="Brand colors"
+        subtitle="Use for designing anything centered around Khan Academy as a brand. Secondary brand colors should only be used for decoration. Expanded brand palette used for illustrations."
+        colors={BrandColors}
+    />
+</ColorPalette>
+
+## Usage
+
+`wonder-blocks-color` exports a set of colors that can be used in your
+components. The colors are exported as constants and can be used like so:
+
+
+```js
+import Color from "@khanacademy/wonder-blocks-color";
+const styles = {background: Color.darkBlue};
+```
+
+### Product
+
+Use only accessible colors against appropriate backgrounds for links, buttons,
+and other navigational elements.
+
+<Swatch
+    name="blue"
+    color={Color.blue}
+    use="text"
+    description="Actionable. Used to indicate actionable items like links and buttons."
+/>
+
+<Swatch
+    name="purple"
+    color={Color.purple}
+    use="icons"
+    description="Used to indicate Mastery."
+/>
+
+<Swatch
+    name="gold"
+    color={Color.gold}
```

Comment:
Did you intentionally not give gold a `use` section?

Reply:
Yep, I based these changes on the previous Styleguidist docs. I think the intention was to leave that part empty b/c it's optional for icons, so they didn't want to state that this would be always the case.

Do you think we could include a valid use for this?

Reply:
Oh okay! That's totally fine. I just wasn't sure if that was on purpose.

I don't work a lot with these colors, so I was just wonder. Green is used for success icons, and purple is used for mastery icons. What _is_ gold used for? Is it used as an accents color? I know it's the crown on the mastery tower; is it commonly used anywhere else?

Reply:
I just checked in Webapp, and it's mostly used for Icons, "warning" states (using it as bg color) and a couple of cases for borders.